src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final UserService userService;
+    private final UserResponseMapper userResponseMapper;
+
+    public AuthenticationController(AuthenticationService authenticationService,
+                                    UserService userService,
+                                    UserResponseMapper userResponseMapper) {
+        this.authenticationService = authenticationService;
+        this.userService = userService;
+        this.userResponseMapper = userResponseMapper;
+    }
+
+    @PostMapping("/register")
+    public UserResponseDto register(UserRequestDto userRequestDto) {
```

Comment:
This method should return void 
![image](https://user-images.githubusercontent.com/32339836/235701523-9b31f36b-a103-4d10-992d-052a00070d71.png)